From 508304dd397be11ec37b6d61f1be4cf60a1ffdae Mon Sep 17 00:00:00 2001
From: Bram Goffings <bramgoffings@gmail.com>
Date: Thu, 21 May 2015 20:45:15 +0200
Subject: [PATCH] Issue #2490932: Undefined property:
 DatabaseConnection_mysql::$sid in node_ds_search_execute() (line 488 of
 ds_search.module)

---
 modules/ds_search/ds_search.module | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/ds_search/ds_search.module b/modules/ds_search/ds_search.module
index fdabe5a..c0b60af 100644
--- a/modules/ds_search/ds_search.module
+++ b/modules/ds_search/ds_search.module
@@ -485,7 +485,7 @@ function node_ds_search_execute($keys = NULL, $conditions = NULL) {
   // Load results.
   $find = $query->execute();
   $results = array();
-  foreach ((array) $find as $item) {
+  foreach ($find as $item) {
     $node = node_load($item->sid);
     $node->entity_type = 'node';
     $node->entity_id = $item->sid;
-- 
2.4.1

